src/commands/createWebApp/createWebApp.ts

```diff
@@ -18,7 +18,7 @@ import { getCreatedWebAppMessage, showCreatedWebAppMessage } from "./showCreated
 import { WebAppStackStep } from "./stacks/WebAppStackStep";
 import { WebAppCreateStep } from "./WebAppCreateStep";
 
-export async function createWebApp(context: IActionContext & Partial<ICreateChildImplContext>, node?: SubscriptionTreeItemBase | undefined, suppressCreatedWebAppMessage: boolean = false): Promise<SiteTreeItem> {
+export async function createWebApp(context: IActionContext & Partial<ICreateChildImplContext>, node?: SubscriptionTreeItemBase | undefined, _nodes?: (SubscriptionTreeItemBase | undefined)[], suppressCreatedWebAppMessage: boolean = false): Promise<SiteTreeItem> {
```

Comment:
I'm going to try to make it so we don't have refactor commands in this way. See https://github.com/microsoft/vscode-azuretools/pull/1246.

We can leave it for now, but once we make progress on https://github.com/microsoft/vscode-azuretools/pull/1246, we can undo these changes.